corehq/apps/hqwebapp/apps.py

```diff
@@ -7,3 +8,35 @@ class HqWebAppConfig(AppConfig):
     def ready(self):
         # Ensure the login signal handlers have been loaded
         from . import login_handlers, signals  # noqa
+        # custom 2FA methods need to be registered on startup
+        register_two_factor_methods()
+
+
+def register_two_factor_methods():
+    from two_factor.plugins.registry import registry
```

Comment:
Is this an import that can be moved to the top of the module? I'd like to conform to [PEP8](https://peps.python.org/pep-0008/#imports) when possible

Reply:
Thanks yeah I moved it up in 2543f363fb65820a1cf26a038e5b90e73aae1c56. When testing locally I got a `AppRegistryNotReady` error when attempting to import the methods in `two_factor_methods.py` at the top of the module because that file imports django_otp, which is not ready when the app ready handler is called here (not quite sure why that is as `django_otp` is included prior to this app, but didn't feel worth digging into more as local imports in an app ready handler are common).